contents/blog/new-vp-nailing-funnels/new-vp-nailing-funnels.md

```diff
@@ -0,0 +1,80 @@
+---
+date: 2021-07-01
+title: How PostHog's New VP Focused The Company On Nailing Funnels In His First Week
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+author: marcus-hyett
+featuredImage: ../../images/blog/new-vp-nailing-funnels/new-vp-nailing-funnels.png
+---
+
+Ramping up fast in any senior product role requires understanding as much context as possible across 3 axes (Market, Product and People). I’ll talk you through my process of going from zero understanding in these three areas, to landing a clear, ambitious long-term strategy across the company in just one week.
+
+## Phase 1: Understanding Context
+
+### Market
+
+**Customer interviews**
+
+We have a wealth of documented in-depth interviews with existing customers and prospects around their experience with PostHog, and I was able to attend a few live interviews in my first week. I read the transcript of every single interview (some multiple times), pulling out key themes and looking beyond what they were asking for to try and work out _why_ they were asking for it. This created a shortlist of critical needs we must solve for to make our customers successful.
+
+**Competitor research**
+
+When you’re new to a market with existing competitors it’s important to understand who they are and what’s unique about their products. I analyzed the marketing and documentation of a number of our competitors to understand our potential gaps and advantages. It’s key not to get hung up on features our competitors have which we don’t - we must focus on building the best solution to the problems our customers have, which might look different to what our competitors offer.
+
+**Analytics**
+
+PostHog is a product analytics tool, so I was able to get set-up and analyse the behavior of our users within a few hours. I focused on breaking down how our existing product is used across different dimensions to understand which of our capabilities are most valuable to our existing customers and have the biggest growth opportunities.
+
+### Product
+
+**Mission**
+
+Posthog’s mission to `"Increase the number of successful products in the world"` was a big reason I joined the company, and that phrase provided context to building out a long-term direction.
```

Comment:
I think we don't need code blocks here? Made it look weird in the post.
```suggestion
Posthog’s mission to "Increase the number of successful products in the world" was a big reason I joined the company, and that phrase provided context to building out a long-term direction.
```

Reply:
That was the original format; James suggested it needed to be 'highlighted'

Reply:
Maybe bold face then?